ui/page/wallet_page.go

```diff
@@ -694,9 +698,13 @@ func (pg *WalletPage) walletAccountsLayout(gtx layout.Context, account *dcrlibwa
 	)
 }
 
-func (pg *WalletPage) backupSeedNotification(gtx layout.Context, listItem *walletListItem) layout.Dimensions {
+func (pg *WalletPage) backupSeedNotification(gtx layout.Context, index int) layout.Dimensions {
 	gtx.Constraints.Min.X = gtx.Constraints.Max.X
 	textColour := pg.Theme.Color.InvText
+	click := pg.backupAcct[index]
+	pointer.Rect(image.Rectangle{Max: gtx.Constraints.Max}).Add(gtx.Ops)
+	click.Add(gtx.Ops)
+	pg.openAcctBackup(gtx, click)
```

Comment:
rather than handling click operations from the layout code, you can create a clickableRow widget that handles the click operations and returns a Clicked(). A clickableRow widget would also be useful in the settings page.